dcap/types/src/quote3.rs

```diff
@@ -367,6 +367,60 @@ impl<'a> TryFrom<&'a [u8]> for CertificationData<'a> {
     }
 }
 
+impl<'a> IntoIterator for &'a CertificationData<'a> {
```

Comment:
I'm thinking with the addition of this, if this is the desired path, the `CertificationData` should get moved out to it's own module, but in a follow on PR that only does that.